### ISSUE TITLE
Failed to handle key=U. Cannot read property 'start' of undefined

### ISSUE THREAD

**Issue by Tig10** [VSCodeVim/Vim#6377]
**To Reproduce**

With VSCodeVim extension installed and enabled in VSCode:

1.  Get into NORMARL mode
2.  Type uppercase 'U'
3.  See error

**VSCodeVim version**: 1.19.3

<details><summary>Stack trace</summary>

```
TypeError: Failed to handle key=U. Cannot read property 'start' of undefined
    at p.goBackHistoryStepsOnLine (c:\.vscode\extensions\vscodevim.vim-1.19.3\src\history\historyTracker.ts:752:51)
    at Re.exec (c:\.vscode\extensions\vscodevim.vim-1.19.3\src\actions\commands\actions.ts:1158:56)
    at Re.execCount (c:\Users\user\.vscode\extensions\vscodevim.vim-1.19.3\src\actions\base.ts:218:20)
    at I.runAction (c:\.vscode\extensions\vscodevim.vim-1.19.3\src\mode\modeHandler.ts:719:20)
    at I.handleKeyAsAnAction (c:\.vscode\extensions\vscodevim.vim-1.19.3\src\mode\modeHandler.ts:650:16)
    at I.handleKeyEvent (c:\.vscode\extensions\vscodevim.vim-1.19.3\src\mode\modeHandler.ts:469:40)
    at runMicrotasks (<anonymous>)
    at process.text (internal/process/task_queues.js:97:5)
    at Object.promise (c:\.vscode\extensions\vscodevim.vim-1.19.3\extensionBase.ts:377:73)
    at Object.runTasks (c:\.vscode\extensions\vscodevim.vim-1.19.3\src\taskQueue.ts:36:26)
```

</details>

**Comment by J-Fields** [comment]
@Tig10 Can you reliably reproduce this?

**Comment by whippingdot** [comment]
I got it just now. I don't know how, but I did. I think I just spammed 'U' or something. I will tell you if I figure out how to do this.

**Comment by Tig10** [comment]
> @Tig10 Can you reliably reproduce this?
Yes I am able to reproduce with `Shift+u` while in NORMAL mode.

**Comment by J-Fields** [comment]
@Tig10 Can you give exact keystrokes for a minimal reproduction, starting from an empty file?

**Comment by Peredery** [comment]
same, after spamming "U"

**Comment by J-Fields** [comment]
I haven't been able to reproduce, but this should've been fixed by 63f41601072c3db238700bf4cb54a1190f86606c
